.github/workflows/ci-coverage-build.yml

```
@@ -1,8 +1,9 @@
 name: Coverage Build
 on:
   workflow_dispatch:
-    branches:
-      - master
+    push:
+      branches:
+        - master
```

Shouldn't the dispatch and push trigger be two separate entries?
```suggestion
    branches:
      - master
  push:
    branches:
      - master
```

---

I think this is fine..

---

we have it like tbis elsewhere too